File: pset1/cash/cash.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>

int main(void)
{
    float dollars;
    int m25, m10, m5, m1, sum = 0;

    do
    {   
        // Request amount entry > 0
        dollars = get_float("Change owed: ");
    }
    while (dollars < 0);
    // Convert to cents
    int cents = round(dollars * 100);
    // Get the amount using "/" and "%"
    sum = cents / 25 + cents % 25 / 10 + cents % 25 % 10 / 5 + cents % 25 % 10 % 5 / 1;

    printf("%i\n", sum);
}

File: pset6/cash/cash.py
from cs50 import get_float


def main():
    dollars = get_positive_float()
    cents = int(round(dollars * 100))
    sum_m = cents // 25 + cents % 25 // 10 + cents % 25 % 10 // 5 + cents % 25 % 10 % 5
    print(sum_m)


def get_positive_float():
    while True:
        a = get_float("Enter dollars: ")
        if a > 0:
            break
    return a


main()